src/app/models/snippet.ts

```diff
@@ -4,47 +4,38 @@ export class Snippet {
   id: string;
```

Comment:
this has been updated, and will need further updates to enable backwards compatibility